packages/ag-charts-community/src/chart/series/cartesian/barSeries.ts

```diff
@@ -369,7 +407,7 @@ export class BarSeries extends AbstractBarSeries<Rect, BarSeriesProperties, BarN
                 midPoint: { x: rect.x + rect.width / 2, y: rect.y + rect.height / 2 },
                 fill,
                 stroke,
-                opacity: 1,
+                opacity,
```

Comment:
Does this impact series highlighting? It maybe that opacity was being applied at the series level, which is why it wasn't set here?

Reply:
Nope. createNodeData isn't called for series highlighting